docs/capability-types.md

```diff
@@ -40,7 +40,7 @@ To make common percentage values more readable, one can use specific keywords to
 * [Pan](#pan) / [PanContinuous](#pancontinuous)
 * [Tilt](#tilt) / [TiltContinuous](#tiltcontinuous)
 * [PanTiltSpeed](#pantiltspeed)
-* [Effect](#effect) / [EffectIntensity](#effectintensity) / [EffectSpeed](#effectspeed) / [EffectDuration](#effectduration)
+* [Effect](#effect) / [EffectParameter](#effectparameter) / [EffectSpeed](#effectspeed) / [EffectDuration](#effectduration)
```

Comment:
Please move EffectParameter behind EffectSpeed and EffectDuration.

- [ ] here
- [ ] in the docs below
- [ ] in the schema
- [ ] in the UI import order
- [ ] anywhere else I have forgotten?